### PR TITLE
Hw 3

### PR DESCRIPTION
**Pull request by andrew221b**

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+// Decompose number. It's my solution
+
+let someNumber = prompt('Enter the number');
+someNumber = String(someNumber);
+let result = someNumber.split('').join(' ');
+alert(result)
+
+
+// Another method with the help of Google. But I don't understand everything is going on with RegEx, to bo honest :)
```
Comment (web-scavenger) [inline assets/js/main.js:18]: It's nice that you are exploring diff solutions, but don't push the horses)

File: assets/js/main.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// 3 strings into 1 line
+
+let userName = prompt('User\'s Name');
+let userAge = prompt('User\'s Age');
+let userCity = prompt('User\'s City');
+
+let userInfo = `${userName} ${userAge} ${userCity}`;
```
Comment (web-scavenger) [inline assets/js/main.js:7]: it will be better to use ```const``` instead of ```let``` as far as we don't change data

File: assets/js/main.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+alert(userInfo);
+
+// Decompose number. It's my solution
+
+let someNumber = prompt('Enter the number');
+someNumber = String(someNumber);
+let result = someNumber.split('').join(' ');
```
Comment (web-scavenger) [inline assets/js/main.js:14]: Do you understand how it works? 
Nice solution)

**Review by web-scavenger — APPROVED** [review]

File: assets/js/main.js (hunk 3 above)
Comment (andrew221b) [reply assets/js/main.js:14]: Yeah, it was pretty obvious for me...to my surprise xD

**Pull request merged**